packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -301,6 +325,15 @@ try {
 		}
 	}
 
+	if (primaryPhp.isFile(remoteAssetListPath)) {
+		const remoteAssetPaths = primaryPhp
+			.readFileAsText(remoteAssetListPath)
+			.split('\n');
+		remoteAssetPaths.forEach((wpRelativePath) =>
```

Comment:
This is done once and stored for the rest of this worker's life. However, we can backfill the missing assets or hotswap the WordPress version. This seems like a cheap operation – can we just do it every time in the 404 handler?

Reply:
That sounds great, but let's make that change in a follow-up PR. Issue [here](https://github.com/WordPress/wordpress-playground/issues/1622).